tests/devapps/XForms/XForms/SettingsPage.xaml.cs

```diff
@@ -133,5 +133,11 @@ private void CreateB2CAppSettings()
             App.RedirectUriOnAndroid = App.RedirectUriB2C;
             App.RedirectUriOnIos = App.RedirectUriB2C;
         }
+
+        private void OnAcquireTokenWithBrokerToggled(object sender, ToggledEventArgs args)
+        {
+            App.UseBroker = args.Value;
```

Comment:
> args.Value [](start = 27, length = 11)

could pass as argument to InitPublicClient instead

Reply:
#! for now.

---
In reply to: [258728699](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/881#discussion_r258728699) [](ancestors = 258728699)